corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -6,17 +6,25 @@
     <button id="scroll-to-bottom" class="btn btn-lg btn-circle btn-primary hidden-md hidden-lg" type="button" aria-label="{% trans 'Scroll to bottom' %}" style="display: none;">
       <i class="fa fa-lg fa-arrow-down" aria-hidden="true"></i>
     </button>
-    <div class="page-header menu-header" id="case-list-menu-header">
-      <% if (sidebarEnabled) {%>
+    <div class="page-header menu-header clearfix" id="case-list-menu-header">
+      <div class="pull-right">
+      <% if (showMap) { %>
+        <button id="hide-map-button" class="btn btn-primary visible pull-right" type="button"
+           aria-expanded="false" aria-controls="module-case-list-map">
```

Comment:
`aria-expanded` should toggle when the map is made visible or hidden (and also default to true if the map is always shown by default). It works automatically for the `show-more` action because Bootstrap handles it as part of `collapse`. Here, you'd probably toggle it manually in`showHideMap()`